### DISCUSSION TITLE
How to merge custom quirks to the repository? My pull request is #3392

### DISCUSSION THREAD

**Discussion by simon3panda** [zigpy/zha-device-handlers#3407] [General]
Hi,
I want to contribute custom quirks to zha-device-handlers project and create a pull request. It seems all conflicts and tests have been done.
![github pull request](https://github.com/user-attachments/assets/f7b985ef-ac94-4664-81c2-d8c559775406)
In the bottom of the pull request page, it shows this message as below. Does it means I don't have the merge access? If so, what can I do to get the access?
![write access](https://github.com/user-attachments/assets/c907d054-76fc-49a4-83b0-a8a6c1da9669)

**Comment by prairiesnpr** [comment]
You can't, you have to wait for a review and then for one of the maintainers to merge.

**Reply by simon3panda** [reply]
Thank you for help!
